library/models.py

```diff
@@ -54,6 +55,11 @@ def __unicode__(self):
     def get_absolute_url(self):
         return reverse('library:book_detail', kwargs={'pk': self.pk})
 
+    @property
+    def index_strings(self):
+        return [self.title, self.isbn, self.authors, self.subtitle,
```

Comment:
For [performances](https://stackoverflow.com/questions/3340539/why-is-tuple-faster-than-list).